src/pynwb/tests/integration/test_ndx_nirs.py

```diff
@@ -78,6 +78,9 @@ def create_fake_nirs_device():
         channels=channels,
         sources=channels.source.table,
         detectors=channels.detector.table,
+        time_delay=4.2,
+        time_delay_width=0.5,
+        additional_parameters="flux capacitor gain = 9000",
```

Comment:
💯

Reply:
But usually it's [over 9000](https://i.kym-cdn.com/entries/icons/original/000/000/056/itsover1000.jpg) 😉